sklearn/metrics/tests/test_pairwise_distances_reduction.py

```diff
@@ -952,6 +953,58 @@ def test_pairwise_distances_radius_neighbors(
     )
 
 
+@pytest.mark.parametrize(
+    "PairwiseDistancesReduction",
+    [PairwiseDistancesArgKmin, PairwiseDistancesRadiusNeighborhood],
+)
+@pytest.mark.parametrize("metric", ["manhattan", "euclidean"])
+def test_memmap_backed_data(
+    global_random_seed,
+    metric,
+    PairwiseDistancesReduction,
+    n_samples=512,
+    n_features=100,
+    dtype=np.float64,
+):
+    # Results must not depend on the datasets writability
+    rng = np.random.RandomState(global_random_seed)
```

Comment:
Is there a commit that test `global_random_seed` for all seeds on the CI using https://github.com/scikit-learn/scikit-learn/pull/23026#issue-1190347858 ?

Also, does this test require `global_random_seed`?

Reply:
> Is there a commit that test global_random_seed for all seeds on the CI using #23026 (comment) ?

No, thus I've pushed dc26da073615b309fd953045d52ddef9e6effa74.

> Also, does this test require global_random_seed?

No, removed in 94c76fbaa7f9169abba8cf12eb6c42c84bc2d550.

Reply:
I pushed [`b089de3` (#23724)](https://github.com/scikit-learn/scikit-learn/pull/23724/commits/b089de367b50a0e4c905745d58e3eb21604faeeb) to trigger the random seeds. [`dc26da0` (#23724)](https://github.com/scikit-learn/scikit-learn/pull/23724/commits/dc26da073615b309fd953045d52ddef9e6effa74) failed because of the "Co-authored-..." in the message.